plugin/pulp/plugin/distributor.py

```diff
@@ -0,0 +1,18 @@
+from pulp.platform.models import RepositoryDistributor
+
+
+class RepositoryDistributor(RepositoryDistributor):
+    """
+    The base RepositoryDistributor object
+    """
+
+    def publish(self):
+        """
+        Perform a publish
```

Comment:
It would be great to emphasize here that the plugin writer should definitely not try to implement their publish logic on this class. They should implement it somewhere else, and just call their workflow from here.